### ISSUE TITLE
Review: 2.4.3 Focus Order (A)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#51]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 2.4.3 considering the mobile research questions (to be completed)

---
If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.
---
Sufficient Techniques for Success Criterion 2.4.3

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    G59: Placing the interactive elements in an order that follows sequences and relationships within the content
    Giving focus to elements in an order that follows sequences and relationships within the content using one of the following techniques:
        C27: Making the DOM order match the visual order
        [PDF3: Ensuring correct tab and reading order in PDF documents ](https://www.w3.org/WAI/WCAG22/Techniques/pdf/PDF3.html)
    Changing a Web page dynamically using one of the following techniques:
        SCR26: Inserting dynamic content into the Document Object Model immediately following its trigger element
        H102: Creating modal dialogs with the HTML dialog element
        [SCR27: Reordering page sections using the Document Object Model ](https://www.w3.org/WAI/WCAG22/Techniques/client-side-script/SCR27.html)

Failures for Success Criterion 2.4.3

    F44: Failure of Success Criterion 2.4.3 due to using tabindex to create a tab order that does not preserve meaning and operability
    F85: Failure of Success Criterion 2.4.3 due to using dialogs or menus that are not adjacent to their trigger control in the sequential navigation order
